doc/source/reference/arrays.nditer.rst

```diff
@@ -115,6 +115,10 @@ context is exited.
     array([[ 0,  2,  4],
            [ 6,  8, 10]])
 
+Prior to version 1.15.0, :class:`nditer` was not a context manager and did not
```

Comment:
Do we need this history? I think it just distracts from the flow of the text. Or at least be explicit why this is here; maybe something like:
```
If you are writing code that needs to support older versions of numpy, note that prior to 1.15, ...
```

Reply:
I agree that this doesn't need to be right there, but I didn't know where else to put it. I added it in because I ran into issues with the syntax in a couple of places. Perhaps a note admonition with the text you suggest?

Reply:
That sounds good. Or use `.. note::` (which perhaps gives it too much emphasis, but it is an actual "note").

Reply:
I can do `**Note:** ...` as a compromise.

Reply:
The particular use case that triggered the comment was IDEOne, which uses version 1.12 or so.